ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -893,6 +893,69 @@ def wait_for_ssh_threads():
                 # verify there are no interface flaps after warm boot
                 self.neigh_lag_status_check()
 
+        if self.reboot_type == 'fast-reboot':
+            self.no_cp_replies = self.extract_no_cpu_replies(self.upper_replies)
+            if self.no_cp_replies < 0.95 * self.nr_vl_pkts:
+                self.fails['dut'].add("Dataplane didn't route to all servers, when control-plane was down: %d vs %d" % (self.no_cp_replies, self.nr_vl_pkts))
+
+    def handle_advanced_reboot_health_check_kvm(self):
```

Comment:
>handle_advanced_reboot_health_check_kvm [](start = 8, length = 39)

The function body is similar to `handle_fast_reboot_health_check`. Reuse code?

Reply:
The logic in kvm test is different from `handle_fast_reboot_health_check`. For example, fast reboot test expects the data plane to stay up for a while after the control plane goes down, this is not expected in kvm. Reusing the code requires a lot of checks for kvm in the function, which is a kind of distraction. Also, separating it into another function would make kvm tests and real switch tests independence. It would be beneficial if we want to add kvm or physical device specific checks in the future, since we only need to change the corresponding functions.